dashboard/src/components/entitypage/elements/PrometheusGraph.js

```diff
@@ -68,15 +69,19 @@ const PrometheusGraph = ({
     (start, end, step, stub) => {
       const startTimestamp = start.getTime() / 1000;
       const endTimestamp = end.getTime() / 1000;
+      step = 5;
       const url = `http://localhost:9090/api/v1/query_range?query=${query}${add_labels_string}&start=${startTimestamp}&end=${endTimestamp}&step=${step}s`;
-      // if (!stub) {
-      //   return Promise.resolve(JSON.parse(sample_query_data));
-      // }
+      if (!remote) {
+        return Promise.resolve(JSON.parse(sample_query_data));
+      }
```

Comment:
Is this just the fallback if the prometheus server cant be reached?

Reply:
yes